resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
```

Comment:
Why not do it in the client? It's easy to do it there and makes no real difference to the events being sent through to have an extra ID. 

We need child events, it is a business requirement. We can't just do counts of things it's too basic. 

Finally, how would you do it down stream? It's not possible to join them up accurately, the code would be hugely complex and it would be inefficient and slow. (Ref: multiple browser tabs, multiple components on a page and trying to fuzzy match based on dates)



> On 28 May 2015, at 22:35, Matt Chadburn <notifications@github.com> wrote:
> 
> In resources/tag_models/event.json:
> 
> > @@ -0,0 +1,29 @@
> > +{
> > +  "tag": {
> > +	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
> > +	"version": "", // Version of the tracking client e.g. "JS v1.2"
> > +	"type": "event", // Type of request (page or event)
> > +	"id": "", // ID of this request, must be unique
> Mmm, as discussed elsewhere, I'd just get rid of child events. If someone want to group the data by user, page, or date sequence, that's all possible later down the line.
> 
> —
> Reply to this email directly or view it on GitHub.
> 

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.